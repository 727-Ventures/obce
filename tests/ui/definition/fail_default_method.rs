#[obce::definition]
pub trait Trait {
    fn default_method() -> u32 {
        123
    }
}

fn main() {}
