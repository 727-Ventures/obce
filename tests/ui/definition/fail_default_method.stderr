error: default implementation is not supported in chains extension
 --> tests/ui/definition/fail_default_method.rs:3:32
  |
3 |       fn default_method() -> u32 {
  |  ________________________________^
4 | |         123
5 | |     }
  | |_____^
